src/plugins/pcr/src/PrimerLibraryWidget.cpp

```diff
@@ -136,11 +143,42 @@ void PrimerLibraryWidget::sl_exportPrimers() {
     exportDialog->exec();
 }
 
+void PrimerLibraryWidget::sl_openTemperatureSettings() {
+    U2OpStatusImpl os;
+    PrimerLibrary* library = PrimerLibrary::getInstance(os);
+    CHECK_OP_UI(os, );
+
+    auto dialog = new TempCalcDialog(this, library->getTemperatureSettings());
+    dialog->open();
+    connect(dialog, &QDialog::finished, this, [this, library](int result) {
```

Comment:
Use a simpler standard code to manage dialogs (see another comment about dialogs above)

Reply:
See [my comment](https://github.com/ugeneunipro/ugene/pull/1114#discussion_r1087581524)